lib/dfe/analytics/event_schema.rb

```diff
@@ -0,0 +1,73 @@
+module DfE
+  module Analytics
+    class EventSchema
```

Comment:
```suggestion
    module EventSchema
```

Why is this a class and not a module?